applications/luci-app-yggdrasil/root/usr/share/rpcd/acl.d/luci-app-yggdrasil.json

```diff
@@ -0,0 +1,10 @@
+{
+        "luci-app-yggdrasil": {
+                "description": "Grant access to LuCI app yggdrasil",
+                "read": {
```

Comment:
Since `yggdrasilctl` provides commands to modify the system state (such as `addPeer`, `removePeer`, `setTunTap` etc.) we should change `read` to `write` here. There will be no functional change in the LuCI app due to this, but it'll be needed in the future for proper permission handling.

Reply:
Thanks for a review. Changed to write.